src/model/CoilSystemCoolingWater.cpp

```diff
@@ -127,50 +152,10 @@ namespace model {
     }
 
     boost::optional<HVACComponent> CoilSystemCoolingWater_Impl::containingHVACComponent() const {
-      // AirLoopHVACUnitarySystem
-      std::vector<AirLoopHVACUnitarySystem> airLoopHVACUnitarySystems = this->model().getConcreteModelObjects<AirLoopHVACUnitarySystem>();
-
-      for (const auto& airLoopHVACUnitarySystem : airLoopHVACUnitarySystems) {
-        if (boost::optional<HVACComponent> coolingCoil = airLoopHVACUnitarySystem.coolingCoil()) {
-          if (coolingCoil->handle() == this->handle()) {
-            return airLoopHVACUnitarySystem;
-          }
-        }
-      }
-
       return boost::none;
     }
 
     boost::optional<ZoneHVACComponent> CoilSystemCoolingWater_Impl::containingZoneHVACComponent() const {
-
-      // ZoneHVACFourPipeFanCoil
-      std::vector<ZoneHVACFourPipeFanCoil> zoneHVACFourPipeFanCoils;
-
-      zoneHVACFourPipeFanCoils = this->model().getConcreteModelObjects<ZoneHVACFourPipeFanCoil>();
```

Comment:
CANNOT be part of a containing ZoneHVACComponent AFAIK

This is from the E+ IDD, there are ZERO possible references.